src/renderer/electron-types.ts

```diff
@@ -19,41 +24,84 @@ export class ElectronTypes {
 
   constructor(
     private readonly monaco: typeof MonacoType,
-    private readonly cacheDir: string,
+    private readonly electronCacheDir: string,
+    private readonly nodeCacheDir: string,
   ) {}
 
   public async setVersion(ver?: RunnableVersion): Promise<void> {
     this.clear();
 
     if (!ver) return;
+
+    await this.setElectronTypes(ver);
+    await this.setNodeTypes(ver.version);
+  }
+
+  public async setElectronTypes(ver: RunnableVersion): Promise<void> {
```

Comment:
```suggestion
  public async setElectronTypes(ver: RunnableVersion): Promise<void> {
```

Looks like this is only called from setVersion(); this could be private